<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsPricingClass</class>
 <widget class="QWidget" name="OptionsPricingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OptionsPricing</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelS">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>S</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelK">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>K</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelt">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>t</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelO">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ïƒ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelr">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>r</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxS">
       <property name="decimals">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxK">
       <property name="decimals">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="doubleSpinBoxT">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxO">
       <property name="decimals">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="doubleSpinBoxR">
       <property name="decimals">
        <number>4</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonCalculate">
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Put:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLCDNumber" name="lcdNumberPut"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Call: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLCDNumber" name="lcdNumberCall"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonClear">
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="OptionsPricing.qrc"/>
 </resources>
 <connections/>
</ui>
